src/wolfio.c

```diff
@@ -151,7 +155,18 @@ static int TranslateIoError(int err)
         return WOLFSSL_CBIO_ERR_CONN_CLOSE;
     }
 
-    WOLFSSL_MSG("\tGeneral error");
+#if defined(_WIN32)
+    strcat_s(errstr, sizeof(errstr), ": ");
+    errstr_offset = strlen(errstr);
+    FormatMessageA(FORMAT_MESSAGE_FROM_SYSTEM | FORMAT_MESSAGE_IGNORE_INSERTS,
+        NULL,
+        err,
+        MAKELANGID(LANG_NEUTRAL, SUBLANG_DEFAULT),
+        (LPSTR)(errstr + errstr_offset),
+        (DWORD)(sizeof(errstr) - errstr_offset),
+        NULL);
+#endif
```

Comment:
Move this into `_WIN32` and restore original call in `#else`. This will reduce the stack use.